doc/faq.rst

```diff
@@ -58,13 +66,29 @@ fundamentally multi-dimensional. If your data is unstructured or
 one-dimensional, stick with pandas.
 
 
-Gotchas
--------
+Why don't aggregations return Python scalars?
+---------------------------------------------
+
+xarray tries hard to be self-consistent: operations on a ``DataArray`` (resp.
+``Dataset``) return another ``DataArray`` (resp. ``Dataset``) object. In
+particular, operations returning scalar values (e.g. indexing or aggregations
+like `mean` or `sum` applied to all axes) will also return xarray objects.
+
+Unfortunately, this means we sometimes have to explicitly cast our results from
+xarray when using them in other libraries. As an illustration, the following
+code fragment
+
+.. ipython:: python
+
+    arr = xr.DataArray([1, 2, 3])
+    pd.Series({'x': arr[0], 'mean': arr.mean(), 'std': arr.std()})
+
+does not yield the pandas DataFrame we expected. We need to do the type
+conversion ourselves:
+
+.. ipython:: python
 
-xarray tries hard to be self-consistent. Operations on xarray objects return
-other xarray objects. In particular, operations like `mean` or `sum` - even
-when applied to all axes - will return xarray objects. Constructing `pandas`
-objects sometimes require explicit casting.
+    pd.Series({'x': arr[0], 'mean': arr.mean(), 'std': arr.std()}).astype(float)
 
```

Comment:
Can you also note `float(arr[0])` and `arr[0].item()` as two others ways to convert scalars to builtin types?